src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
```

Comment:
Instead of writing left indentation and then right indentation, you can use internal indentation for the header
```suggestion
```